servicecontrol/plugins/index.md

```diff
@@ -13,15 +13,32 @@ NOTE: When ServiceControl is introduced into an existing environment the standar
 
 Plugins collect information from NServiceBus and are deployed together with each NServiceBus endpoint. These plugins are optional from the perspective of the NServiceBus framework itself (they are not required by the endpoint), but they are required in order to collect the information that enables ServiceControl (and its clients) to provide the relevant functionality for each plugin.
 
+## Functionalities
 
-## Installing and Deploying
+ServiceControl provide monitoring by analyzing configured audit and fault queues. Based on audit and fault messages it can extract information like: endpoint names, queue names, exception stack traces etc. Based on that information ServiceControl stores them in built-in database and exposes them to 3rd party tools.
+
+Plugins allow to extend given functionalities by providing additional data such as:
+ - [Heartbeat Plugin](heartbeat.md) - report if endpoint has connectivity with ServiceControl by sending heartbeat messages.
+ - [Saga Plugin](saga-audit.md) - report saga related data to allow for better analysis.
+ - [Custom Check Plugin](custom-checks.md) - allow for writing custom checks and sending information about its failure or success.
+
+## Configuring endpoint to be monitored by ServiceControl
+
+To allow for ServiceControl to monitor endpoints messages the following steps should be followed:
+
+1. ServiceControl should be [installed](/servicecontrol/installation.md) and at least one instance should be configured using the same Transport as is used by endpoints
+2. For every endpoint that messages should be monitored by ServiceControl [configure audits](/nservicebus/operations/auditing.md#configuring-auditing). The audit queue should match audit queue configured for ServiceControl.
```

Comment:
>For every endpoint that messages should be monitored by ServiceControl [configure audits](/nservicebus/operations/auditing.md#configuring-auditing). The audit queue should match audit queue configured for ServiceControl.

For every endpoint that is being monitored by ServiceControl, [configure the endpoint for auditing](/nservicebus/operations/auditing.md#configuring-auditing). Make sure that the audit queue is the same as the audit queue that ServiceControl is configured with.